tests/v2/test_waku_store.nim

```diff
@@ -446,6 +448,49 @@ suite "Waku Store":
     ## Cleanup
     await allFutures(clientSwitch.stop(), serverSwitch.stop())
 
+  asyncTest "handle ephemeral messages":
```

Comment:
I see what you did here 😁 

I feel that this is overkill. The goal of the test should be to test the `if msg.ephemeral` statement in the `protocol.nim` file.

What I would do is something simpler:

1. Create a `StoreQueueRef` instance.
```nim
let store = StoreQueueRef.init(...)
```
2. Create a `WakuStore` instance passing the `StoreQueueRef` instance as the `store` parameter.
```nim
let proto = WakuStore.init(..., store=store)
```
3. Handle five messages with ephemeral and non-ephemeral flag:
```nim
let msgList = @[
  fakeWakuMessage(ephemeral = false),
  fakeWakuMessage(ephemeral = true),
  fakeWakuMessage(ephemeral = true),
  fakeWakuMessage(ephemeral = true),
  fakeWakuMessage(ephemeral = false),
]

for msg in msgList:
  await proto.handleMessage(DefaultPubsubTopic, msg)
```
5. Assert that the number of messages in the `StoreQueueRef`:
```nim
check:
  store.len == 2
```

Reply:
Addressed in [5c542e3](https://github.com/status-im/nwaku/pull/1119/commits/5c542e3f6b0ee7ebbbc2022e5df278dc3a5fe5cf)